requirements.txt

```diff
@@ -1,37 +1,37 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.6
```

Comment:
NB (but you probably know): Python 3.6 support [ends at the end of 2021](https://devguide.python.org/#status-of-python-branches).